src/Microsoft.Azure.WebJobs.Host/Bindings/BindingTemplateExtensions.cs

```diff
@@ -74,7 +74,7 @@ private static void ValidateContractCompatibility(IEnumerable<string> parameterN
 
                     if (!bindingDataContract.ContainsKey(parameterName))
                     {
-                        throw new InvalidOperationException(string.Format(CultureInfo.CurrentCulture, "No binding parameter exists for '{0}'.", parameterName));
+                        throw new InvalidOperationException(string.Format(CultureInfo.CurrentCulture, "No binding parameter exists for '{0}' on the trigger input. For binding to custom properties, see https://go.microsoft.com/fwlink/?linkid=864355", parameterName));
```

Comment:
Something like the following is technically more correct (though wordy):  "Unable to resolve binding parameter '{0}'. Binding expressions must map to either a value provided by the trigger or a property of the value the trigger is bound to, or must be a system binding expression (e.g. sys.randguid, sys.utcnow).

I don't think we should link to the Functions specific doc you linked to.